contactus/templates/contactus/contact.html

```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 
-{% block title %}Contact Us{% endblock %}
+{% block title %}Contact Us | {% endblock %}
```

Comment:
The `<title>` element looks okay as it is for this page: e.g. `<title>EconPractice: Contact Us</title>` - what is the need for this change?